### PR TITLE
[Feature] [EDT-1089] Support setting image variable connector options

### PR DESCRIPTION
**Pull request by Dvergar**
Related ticket: https://chilipublishintranet.atlassian.net/browse/EDT-1089

---

New `SDK.variable` methods:

`SDK.variable.setImageVariableConnector` now returns the new connector id.
`SDK.variable.getImageVariableConnectorId` has been added.
`SDK.connector.getOptions` has been added.
`SDK.connector.getMappings` has been added.

_**NOT** a breaking-change_

### CODE REVIEW

File: src/controllers/ConnectorController.ts [hunk 1]
```diff
@@ -96,6 +96,26 @@ export class ConnectorController {
         return res.getConnectorState(id).then((result) => getEditorResponseData<ConnectorState>(result));
     };
 
+    /**
+     * Gets the options from the connector.
```
Comment (NicolaVerbeeck) [inline src/controllers/ConnectorController.ts:100]: Flipped comments?

**Review by pietervp — DISMISSED** [review]

**Review by abdelhalimkhouas — APPROVED** [review]

**Review by Matthiee — APPROVED** [review]

**Review by pkgacek — DISMISSED** [review]

File: src/controllers/VariableController.ts [hunk 2]
```diff
@@ -228,13 +228,23 @@ export class VariableController {
...
+
     /**
      * This method sets the image variable connector. Setting a connector will
      * automatically remove the assetId linked to the connector if present.
      * If a connector was the source of the variable, it will be unregistered.
-     * @param id The ID of the image variable to update
-     * @param registration registration object containing all details about the connector
-     * @returns
+     * @param id The id of the image variable to update
+     * @param connectorId The new id of the connector
```
Comment (pietervp) [inline src/controllers/VariableController.ts:246]: The docs are updated, bur the method still accepts a `ConnectorRegistration` object? I believe this should also be updated to pas an id?

Comment (Dvergar) [reply src/controllers/VariableController.ts:246]: Thanks for noticing this, probably a bad conflict resolution of me.

(it's actually the other way around)

**Review by Matthiee — APPROVED** [review]

**Review by brapoprod — APPROVED** [review]
lgtm

**Pull request merged**